app/models/avoid_posting.rb

```diff
@@ -96,7 +97,19 @@ def search(params)
       q = q.attribute_matches(:staff_notes, params[:staff_notes])
       q = q.where_user(:creator_id, :creator, params)
       q = q.where("creator_ip_addr <<= ?", params[:ip_addr]) if params[:ip_addr].present?
-      q.apply_basic_order(params)
+      case params[:order]
+      when "artist_name", "artist_name_asc", nil
```

Comment:
Why is the default order alphabetical?
That's not the case anywhere else on the site.